### PR TITLE
jakarta updates

### PR DESCRIPTION
**Pull request by denis-yuen**
Update along the lines of https://github.com/dockstore/swagger-java-discourse-client/pull/15 
Will need manual testing, seems like the automated tests for zenodo are not up to snuff

Corrected some endpoint descriptions in openapi (yes, its silly but true, the ending slash is required and leads to ElasticSearch results)

### CODE REVIEW

**Comment by denis-yuen** [comment]
Guess it's time to get this in for some manual testing

**Comment by denis-yuen** [comment]
(actually, think this needs more work)

**Comment by denis-yuen** [comment]
> (actually, think this needs more work)

done

**Review by coverbeck — COMMENTED** [review]
> Corrected some endpoint descriptions in openapi (yes, its silly but true, the ending slash is required and leads to ES results)

What are ES results? 

And are sure the API will work with the trailing slashes?

**Comment by denis-yuen** [comment]
> > Corrected some endpoint descriptions in openapi (yes, its silly but true, the ending slash is required and leads to ES results)
> 
> What are ES results?
> 
> And are sure the API will work with the trailing slashes?

ElasticSearch, like our search API

For the trailing slashes, in fact, it only works with trailing slashes.  i.e. it was broken before

```
$ curl https://zenodo.org/api/licenses
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>Redirecting...</title>
<h1>Redirecting...</h1>
<p>You should be redirected automatically to target URL: <a href="https://zenodo.org/api/licenses/">https://zenodo.org/api/licenses/</a>.  If not click the link.

$ curl https://zenodo.org/api/licenses/
{"aggregations":{},"hits":{"hits":[{"created":"2018-11-22T14:50:27.019476+00:00","id":"0BSD","links":{"self":"https://zenodo.org/api/licenses/0BSD"},"metadata":{"$schema":"http://zenodo.org/schemas/licenses/license-v1.0.0.json","domain_content":false,"domain_data":false,"domain_software":false,"family":"","id":"0BSD","maintainer":"","od_conformance":"not reviewed", ...
```

**Comment by coverbeck** [comment]
> ElasticSearch, like our search API

OK, I thought of that, but then couldn't see how an openapi path definition would effect our ES and thought it was something else. Will look more closely again.

**Review by coverbeck — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -400,7 +393,66 @@
                     </execution>
                 </executions>
             </plugin>
-
+<plugin>
```
Comment (kathy-t) [inline pom.xml]: Nit: wrong tabbing

**Review by kathy-t — APPROVED** [review]

**Review by svonworl — APPROVED** [review]

**Pull request merged**